### ISSUE TITLE
AAPT: error: duplicate value for resource 'attr/button_style' with config ''.

### ISSUE THREAD

**Issue by mhrjnamar** [khalti/khalti-android#19]
After integrating sdk in the project build fails with  following error

/Users/amar/.gradle/caches/transforms-2/files-2.1/5b02e9fabdc6d3f8a2b35986c0906017/jetified-khalti-android/res/values/values.xml:144:5-154:25: AAPT: error: duplicate value for resource 'attr/button_style' with config ''.
   

it points to this resource
 <declare-styleable name="khalti">
        <attr format="string" name="text"/>
        <attr format="enum" name="button_style">
            <enum name="basic" value="-1"/>
            <enum name="khalti" value="0"/>
            <enum name="e_banking" value="1"/>
            <enum name="m_banking" value="2"/>
            <enum name="sct" value="3"/>
            <enum name="connect_ips" value="4"/>
        </attr>
    </declare-styleable>

**Comment by ishowrunes** [comment]
You might have another library with the same attribute name. Can you check what other library has that attribute?

**Comment by iamnabink** [comment]
> You might have another library with the same attribute name. Can you check what other library has that attribute?

@Ishwor-Shrestha Stock in same issue, what could be the solution for this issue? Searched for similar attribute but could not help myself, Any solution please?

**Comment by ishowrunes** [comment]
Did you verify that other libraries are not using the same attribute name?

**Comment by mhrjnamar** [comment]
@Ishwor-Shrestha yeah we had verified that Khalti SDK and Amplify SDK are using the same attribute name. I was provided an offline SDK a year earlier. I see this issue still persists and causing problems in Khalti SDK implementation.
